free-programming-books-zh.md

```diff
@@ -740,3 +741,11 @@
 ### WebAssembly
 
 * [C/C++面向WebAssembly编程](https://github.com/3dgen/cppwasm-book)
+
+
+### TypeScript
+
+* [TypeScript 英文官网](https://www.typescriptlang.org)
+* [TypeScript 中文网](https://www.tslang.cn)
+* [TypeScript 入门教程](http://www.runoob.com/w3cnote/getting-started-with-typescript.html)
+* [TypeScript Deep Dive 中文版](https://github.com/jkchao/typescript-book-chinese)
```

Comment:
Move it to

![image](https://user-images.githubusercontent.com/25427352/52986570-f4b4a780-3432-11e9-9470-784469bfa6c7.png)